### ISSUE TITLE
Internationalization support

### ISSUE THREAD

**Issue by yemikudaisi** [bolorundurowb/moment.net#9]
@bolorundurowb I think its time you consider i18n support, such that the language used for display of a relative time will be based on the ``DateTime``'s ``CultureInfo``

So  for ``CultureInfo("en-US")`` or ``InvarianCulture`` the relative time appears as **8 years ago** and for ``CultureInfo("fr-FR")``, it appears as **il y a 8 ans** (in French, as expected).

**Comment by bolorundurowb** [comment]
@yemikudaisi I agree with you, i18n was always going to be a crucial part of making this project broadly accepted and used. After getting method parity with `moment.js` to an acceptable level, then I'd focus on that.

**Comment by bolorundurowb** [comment]
But as usual, this is an open source project for a reason. if that's not cool with you, you can get started on it and I'll gladly review and merge

**Comment by alegaio7** [comment]
Hello! I added a pull request with support for localization. Tests status is "pass", so now I'm only waiting for approval.
Thanks!

**Comment by bolorundurowb** [comment]
Thanks for the input @alegaio7 . Your PR has been reviewed and merged and would be going out in the next release. 

To the wider community, we are looking for more translations, contributions are welcome

**Comment by bolorundurowb** [comment]
I'll close this ticket, not because internationalisation is done but there is progress on it
